src/ptracer.rs

```diff
@@ -118,6 +118,22 @@ impl Tracee {
         Self { pid, pending, stop, _not_send }
     }
 
+    pub fn pid(&self) -> Pid {
+        self.pid
+    }
+
+    pub fn pending_signal(&self) -> Option<Signal> {
+        self.pending
+    }
+
+    pub fn set_pending_signal(&mut self, pending: impl Into<Option<Signal>>) {
```

Comment:
This should only be permitted when `stop == SignalDelivery`.

Leave this impl for now; we'll follow up when addressing #135.